spec/timezone.html

```diff
@@ -71,19 +61,15 @@ <h1>CanonicalizeTimeZoneName ( _timeZone_ )</h1>
         If an ECMAScript implementation does not include the ECMA-402 API the following specification of the CanonicalizeTimeZoneName abstract operation is used.
       </p>
       <p>
-        The abstract operation CanonicalizeTimeZoneName takes argument _timeZone_, a String value, and returns the canonical and case-regularized form of _timeZone_.
-      </p>
-      <p>
-        CanonicalizeTimeZoneName may only be called with a _timeZone_ parameter that is a valid time zone name according to IsValidTimeZoneName.
-      </p>
-      <p>
-        The minimum implementation of CanonicalizeTimeZoneName, supporting only the *"UTC"* time zone, performs the following steps when called:
+        The abstract operation CanonicalizeTimeZoneName takes argument _timeZone_, a String value that is a valid time zone name as verified by IsValidTimeZoneName, and returns the canonical and case-regularized form of _timeZone_.
       </p>
 
       <emu-alg>
         1. Assert: Type(_timeZone_) is String.
         1. Assert: ! IsValidTimeZoneName(_timeZone_) is *true*.
-        1. Return *"UTC"*.
+        1. Let _tzUpper_ be the String value that is the same as _timeZone_ except that each occurrence of a lowercase ASCII letter code unit (0x0061 through 0x007A, inclusive) has been replaced with its uppercase analog (0x0041 through 0x005A, inclusive).
+        1. If _tzUpper_ is *"UTC"*, return *"UTC"*.
```

Comment:
It would also be possible to remove the ECMA-402 override of CanonicalizeTimeZoneName by including the tzdb steps and the "UTC" overrides here in ECMA-262, but I don't know if that is considered appropriate.